### ISSUE TITLE
Can't place baby down when it's ready to grow

### ISSUE THREAD

**Issue by LilSmoreKnox** [Luke100000/minecraft-comes-alive#517]
I was playing with my friend and decided to have a kid with one villager and named the baby Naveli, when I saw that the parents were mixed up. For an example, I am a female and had the villager as a male, it showed the villager as the mother, and me as the father. Then I've tried placing down Naveli when it said ready to grow. I had the baby in my hand and right clicked and it would not let me place it. I thought it was because I was on multiplayer so I went to a survival world and tried it again, I still couldn't place the baby anyone know why?

**Comment by Luke100000** [comment]
Please add the version, mod loader and latest.log

**Comment by LilSmoreKnox** [comment]
I'm using Minecraft 1.19.3 verison, Forge  44.0.4.0 and im not sure where to find the lastest log..

**Comment by kristibektashi** [comment]
> I'm using Minecraft 1.19.3 verison, Forge 44.0.4.0 and im not sure where to find the lastest log..

%appdata%\\.minecraft\logs\latest.log (if you use Windows). However if you have opened another instance of the game since reproducing the issue (for example vanilla or a different version) you should reproduce the issue again and then send the latest.log

**Comment by LilSmoreKnox** [comment]
> > I'm using Minecraft 1.19.3 verison, Forge 44.0.4.0 and im not sure where to find the lastest log..
> 
> %appdata%\.minecraft\logs\latest.log (if you use Windows). However if you have opened another instance of the game since reproducing the issue (for example vanilla or a different version) you should reproduce the issue again and then send the latest.log

Thank you so very much!

**Comment by LilSmoreKnox** [comment]
> Please add the version, mod loader and latest.log

I'm using Minecraft 1.19.3 verison, Forge 44.0.4.0 and The lastest.logg.. 
[latest.log](https://github.com/Luke100000/minecraft-comes-alive/files/10376805/latest.log)

**Comment by Luke100000** [comment]
Thanks, both of you :) However, the log seems very short, did you try to place the baby during that run?

**Comment by LilSmoreKnox** [comment]
> Thanks, both of you :) However, the log seems very short, did you try to place the baby during that run?

Yeah I have, i even tried again like 1 hour ago, and it still didnt let me try
